instance/prod_config.py.default

```diff
@@ -53,3 +53,11 @@ WHITELISTED_IDS = []
 # Admins will have extra access to create "public" teams, and if ADMINS_ACCESS_ALL_MATCHES
 # is set, they can access admin info for all matches (can pause, cancel, etc.) ANY match.
 ADMIN_IDS = []
+
+# The spectator value is for admins, observers, and other personnel (including coaches)
```

Comment:
Coaches should not be in a spectator list. They should be included in the teams like any player.

Reply:
i removed the wrong part in the description